src/chrome/content/rules/additifs-alimentaires.net.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="additifs-alimentaires.net">
+	<target host="additifs-alimentaires.net" />
+	<target host="www.additifs-alimentaires.net" />
```

Comment:
`<target host="joomla.additifs-alimentaires.net" />`

Reply:
I'd actually prefer we don't add internal subdomains. We've had complaints from domain owners about this in multiple instances in the past.

There is almost no chance a user will visit it anyways so there is not real risk to drop it IMO.

@Harest, could you revert that please?

Reply:
Damn i just saw that by luck @Bisaloo, i had no notification. I'll remove it right now.